src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30)
```

Comment:
```suggestion
                .filter(age -> age >= 30 && age <= 45)
```
조건을 판별하는 우선순위가 없다면 요렇게 한번에 처리해볼수도 있을 것 같습니다!